src/main/resources/data/job-profiles.json

```diff
@@ -16,6 +16,16 @@
       "retail"
     ],
     "permissions": [
+      {
+        "businessFunction": "Manage Pockets",
```

Comment:
what kind of users should have this permission? because we have the retail-banking specific job profile json file as well.

Reply:
@lehphyro Could you answer which user have this permission?

Reply:
Every retail banking user needs to be able to use Pockets.

Reply:
I will do that.